froide/foirequest/api/serializers.py

```diff
@@ -310,25 +348,13 @@ def get_file_url(self, obj):
 
 
 class FoiAttachmentTusSerializer(serializers.Serializer):
-    message = serializers.IntegerField()
+    message = serializers.HyperlinkedRelatedField(
+        view_name="api:message-detail",
+        lookup_field="pk",
+        queryset=FoiMessage.objects.all(),
+    )
     upload = serializers.CharField()
 
-    def validate_message(self, value):
-        writable_requests = get_write_queryset(
-            FoiRequest.objects.all(),
-            self.context["request"],
-            has_team=True,
-            scope="upload:message",
-        )
```

Comment:
what happened to this permission-checking / validation? Why is it not needed anymore?